gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/GradlePluginTest.kt

```diff
@@ -114,6 +115,32 @@ class GradlePluginTest : GradlePluginTestBase() {
         }
     }
 
+    @Test
+    fun testWebJsWasm() = with(
+        testProject(
+            "application/webJsWasm",
+            testEnvironment = defaultTestEnvironment.copy()
+        )
+    ) {
+        gradle(":composeApp:composeWebCompatibilityDist").checks {
+            check.taskSuccessful(":composeApp:composeWebCompatibilityDist")
+
+            file("./composeApp/build/dist/web/productionExecutable").apply {
+                checkExists()
+                assertTrue(isDirectory)
+                val distributionFiles = listFiles()!!.map { it.name }.toSet()
+
+                assertTrue(distributionFiles.any { it.endsWith(".wasm") })
+
+                assertContentEquals(distributionFiles.filter { !it.endsWith(".wasm") }, setOf(
```

Comment:
`.wasm` is expected too. Can it be included in the check? 
I remember its names is not persistent. Maybe checking that .wasm files are there.

Reply:
@eymar we are checking that wasm is present but unfortunately we can not know exact names - well, we can everything but it's tricky